### ISSUE TITLE
Find all objects in container update

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#4841]
I have temporarily excellent public library internet and AC and thought I'd try again to test where we are in terms of being able to download 1) the positions of all items in a container and 2) a list of all catalog items in a container using the following tools:

![image](https://user-images.githubusercontent.com/14808196/179847534-ac814275-d90d-4a0b-9b3a-f1aca7933425.png)

 1) Using Container Contents CSV only - I was able to get a download of part locations -YAY!!! - except the search only returns the first 20000 parts.
This freezer (DGR12748) should have up to 40,000 cryovials.  I seem to remember some comment before @dusty and I left for travel this summer that he was limiting the number of search results to make the tool work. Any way to increase that to 40,000 - or even better, to allow incremental search, e.g. the first 20,000 - then the second 20,000, then the third 20,000 etc? That way we could get all the parts no matter how large the container by doing incremental downloads?

2) See all collection objects in this container worked great, for three separate freezers, up to 16,248 cataloged items. 
Wondering how many of the problems I've been having with this tool are UNM-related . . . .

In summary - there is progress. BUT - we still haven't achieved ultimate goal of being find the locations of all objects in a container of 30,000 -100,000 objects - an ultralow freezer or a cryotank (Not to mention - See all collection objects in my collection . . . ) At least the former is absolutely essential for me to be able to use Arctos to perform the functions of my job. I need to be able to get a download of everything in a freezer at a given time. Is this possible? If not, what can we do to make it possible? 

Priority: Critical.

**Comment by campmlc** [comment]
Update on See all Collection Objects - I was able to download all objects in a cryotank, and download and map all objects in my ULT freezer room!! Fantastic improvement whatever happened -enormous help, thank you!
So now the remaining issue is to be able to download the part locations - and sequential download or an emailed file would be fine.

**Comment by dustymc** [comment]
I think there are about a dozen of these Issues, they all look about the same to me, not sure why this is open.

Is there some reason we need multiple container downloads? I think they could be merged into one (eg by adding a 'parts and catalog record' column(s) to the contents download).
